### PR TITLE
Add tooltips to extension navbar

### PR DESCRIPTION
**Pull request by InspectorDeno**
Adds clarity to the extension details by adding tooltips, as mentioned in #47586.
extension-navbar-tooltip

### CODE REVIEW

File: src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts [hunk 1]
```diff
@@ -99,10 +99,14 @@ class NavBar {
 		this.actionbar = new ActionBar(element, { animated: false });
 	}
 
-	push(id: string, label: string): void {
+	push(id: string, label: string, tooltip: string): void {
 		const run = () => this._update(id);
 		const action = new Action(id, label, null, true, run);
 
+		if (!action.tooltip) {
```
Comment (sandy081) [inline src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts]: I do not think you need to check for existing tooltip. You can set it to given tooltip always

Comment (InspectorDeno) [reply src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts]: You're right. No need for that then :)

What do you think of the captions as 
`Details` -> `Extension details`
`Contributions` -> `Extension contributions to VS Code editor`
`Changelog` -> `Extension changelog`
`Dependencies` -> `Extension dependencies`

Got a comment and suggestions in #47586 about them being a bit "automatically generated" wich I guess is true. But they're also informative, which was the issue to begin with.
Thoughts?

Comment (sandy081) [reply src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts]: Tooltips can be informative.. I think it makes sense to accept the suggestions from there.

**Comment by InspectorDeno** [comment]
Check for existing tooltip is removed. Captions changed to:
`Details` -> `Extension details, rendered from the extension's 'README.md' file`
`Contributions` -> `Extension contributions to VS Code editor`
`Changelog` -> `Extension update history, rendered from the extension's 'CHANGELOG.md' file`
`Dependencies` -> `Extension dependencies, lists other extensions this extension depends on`

**Comment by sandy081** [comment]
LGTM

Thanks @InspectorDeno

**Pull request merged**